test/mysql/CursorTest.chpl

```diff
@@ -0,0 +1,125 @@
+use DatabaseCommunicator;
+use DatabaseCommunicator.QueryBuilder;
+use UnitTest;
+use MySQL;
+
+proc cursorOpenExecuteTest(test: borrowed Test) throws {
+  var conHandler = ConnectionHandler.ConnectionHandlerWithConfig(MySQLConnection, "dbconfig.toml");
+  var cursor = conHandler.cursor();
+
+  cursor.execute(new Statement("SELECT * FROM sample"));
+
+  cursor.close();
+  conHandler.close();
+}
```

Comment:
There are no assert statements. What are you testing here?

Reply:
I'd like to test if there cursor opens and executes the statement successfully and if it throws an error on wrong SQL syntax. But there's no way to test an exception yet using the UnitTest library hence this function only opens a cursor and executes a statement for now. I'll add a TODO for this in here.

Reply:
Okay, but can you test the successful ones? If yes, please add.

Reply:
Alright, I think I can check that by using a variation of SQL statements (like inserting a row and checking for it), or some other way.